Dockerfile

```
@@ -0,0 +1,85 @@
+FROM python:3.8-slim-buster as base
+
+ENV PYTHONFAULTHANDLER=1 \
+  PYTHONUNBUFFERED=1 \
+  PYTHONHASHSEED=random \
+  PIP_NO_CACHE_DIR=off \
+  PIP_DISABLE_PIP_VERSION_CHECK=on \
+  PIP_DEFAULT_TIMEOUT=100 \
+  POETRY_VERSION=1.0.1 \
+  TRELLO_BOARD=$TRELLO_BOARD \
+  TRELLO_KEY=$TRELLO_KEY \
+  TRELLO_TOKEN=$TRELLO_TOKEN
+
+# System deps:
+RUN pip install "poetry==$POETRY_VERSION"
+
+# Copy only requirements to cache them in docker layer
+WORKDIR /code
+COPY poetry.lock pyproject.toml /code/
+
+# Project initialization:
+RUN poetry config virtualenvs.create false \
+  && poetry install --no-interaction --no-ansi
+
+#------------#
+# PRODUCTION #
+#------------#
+
+# Setting up the production image
+FROM base as production
+
+# Creating folders, and files:
+COPY . /code
+
+# Kick off the production app using gunicorn
+ENTRYPOINT poetry run gunicorn -e TRELLO_BOARD=$TRELLO_BOARD -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN --bind 0.0.0.0:8000 'todo_app:create_app()'
```

```suggestion
ENTRYPOINT poetry run gunicorn --bind 0.0.0.0:8000 'todo_app:create_app()'
```

I think you got confused with the `-e` option of `docker run`. Environment variables that are accessible here will already be accessible by the python process.